<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalShortcutTarget</class>
 <widget class="QDialog" name="GlobalShortcutTarget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Whisper Target</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="qlTarget">
     <property name="text">
      <string>Shout/Whisper to:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="qswStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="qwSelectionPage">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="qgbSelection">
         <property name="title">
          <string>Current selection</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="qlWhenShouting">
            <property name="text">
             <string>When shouting to channel:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QCheckBox" name="qcbShoutLinks">
               <property name="toolTip">
                <string>The whisper will also be transmitted to linked channels.</string>
               </property>
               <property name="text">
                <string>Shout to linked channels</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="qcbShoutSubchans">
               <property name="toolTip">
                <string>The whisper will also be sent to the subchannels of the channel target.</string>
               </property>
               <property name="text">
                <string>Shout to subchannels</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="qwUserPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="qgbUser">
         <property name="title">
          <string>List of users</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" rowspan="4">
           <widget class="QListWidget" name="qlwUsers"/>
          </item>
          <item row="0" column="1">
           <widget class="MUComboBox" name="qcbUser"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="qpbAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="qpbRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>94</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="qwChannelPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="qgbChannel">
         <property name="title">
          <string>Channel Target</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="qtwChannels">
            <property name="indentation">
             <number>10</number>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">Name</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="qlGroup">
              <property name="text">
               <string>Restrict to Group</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="qleGroup">
              <property name="toolTip">
               <string>If specified, only members of this group will receive the whisper.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="qcbLinks">
              <property name="toolTip">
               <string>The whisper will also be transmitted to linked channels.</string>
              </property>
              <property name="text">
               <string>Shout to Linked channels</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="qcbChildren">
              <property name="toolTip">
               <string>The whisper will also be sent to the subchannels of the channel target.</string>
              </property>
              <property name="text">
               <string>Shout to subchannels</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="qgbModifiers">
     <property name="title">
      <string>Modifiers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="qcbForceCenter">
        <property name="toolTip">
         <string>Do not send positional audio information when using this whisper shortcut.</string>
        </property>
        <property name="text">
         <string>Ignore positional audio</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="qcbTarget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Current selection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>List of users</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="qbbButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MUComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/MUComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>qlwUsers</tabstop>
  <tabstop>qcbUser</tabstop>
  <tabstop>qpbAdd</tabstop>
  <tabstop>qpbRemove</tabstop>
  <tabstop>qtwChannels</tabstop>
  <tabstop>qleGroup</tabstop>
  <tabstop>qcbLinks</tabstop>
  <tabstop>qcbChildren</tabstop>
  <tabstop>qcbForceCenter</tabstop>
  <tabstop>qbbButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>GlobalShortcutTarget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>GlobalShortcutTarget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
